inference-engine/src/mkldnn_plugin/nodes/mkldnn_conv_node.cpp

```diff
@@ -47,7 +48,20 @@ MKLDNNConvolutionNode::MKLDNNConvolutionNode(const std::shared_ptr<ngraph::Node>
         IE_THROW(NotImplemented) << errorMessage;
     }
 
-    isPrimitivesPriorityDefined = op->get_rt_info().count("PrimitivesPriority") != 0;
+    internalBlobDesc.emplace_back([&](primitive_desc_iterator &primitive_desc_it, size_t idx) -> MKLDNNMemoryDesc {
```

Comment:
We have to update internal descs only for wino case.

Reply:
done